.storybook/previewMainTemplate.ejs

```diff
@@ -15,7 +15,7 @@
       <link href="./assets/v4/latest/css/qg-noscript.css" rel="stylesheet" type="text/css" media="all">
     </noscript>
         <link href="./assets/v4/latest/css/qg-main.css" rel="stylesheet" type="text/css" media="all">
-    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.6.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```

Comment:
Have you gone through the 3.x to 3.6 upgrade doc's and see if there is any breaking changes we need to keep ontop of?

https://jquery.com/upgrade-guide/3.5/

Reply:
The change in behaviour for self-closing tags should be tested.